tests/handlers/test_freshmaker.py

```diff
@@ -77,6 +77,50 @@ def test_event_to_building(mock_koji_cs):
     assert build.state == 0
 
 
+@mock.patch('koji.ClientSession')
+def test_event_to_complete(mock_koji_cs):
+    """Test the Freshmaker handler when it receives an event to complete message."""
+    mock_koji_session = mock.Mock()
+    mock_koji_session.getTaskResult.side_effect = [
+        {
+            'koji_builds': ['710916']
+        },
+        {
+            'koji_builds': ['123456']
+        },
+        {
+            'koji_builds': ['234567']
+        }
+    ]
+    mock_koji_cs.return_value = mock_koji_session
+    event = FreshmakerEvent.get_or_create({
+        'id_': '2194',
+        'state': 2,
```

Comment:
This state should be `1` since we are trying to mimic the state of the Freshmaker event in Neo4j before the message was processed.